src/TuDa.CIMS.Api/Database/CIMSDbContext.cs

```diff
@@ -5,14 +5,35 @@ namespace TuDa.CIMS.Api.Database;
 
 public class CIMSDbContext : DbContext
 {
+    #region AssetItem
+
     public DbSet<AssetItem> AssetItems { get; init; }
     public DbSet<Consumable> Consumables { get; init; }
     public DbSet<Substance> Substances { get; init; }
     public DbSet<GasCylinder> GasCylinders { get; set; }
     public DbSet<Chemical> Chemicals { get; init; }
     public DbSet<Solvent> Solvents { get; set; }
+
+    #endregion
+
```

Comment:
What is #region for?

Reply:
Region is for the IDE, so you can collapse them and the Context is not so cluttered